Assets/Scripts/UIScripts/AnswerPanel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class AnswerPanel : MonoBehaviour
+{
+	private MultipleChoicePanelEditor multipleChoicePanelEditor;
+
+	void Start()
+	{
+		multipleChoicePanelEditor = transform.root.gameObject.GetComponent<MultipleChoicePanelEditor>();
+		transform.GetComponentInChildren<Toggle>().group = transform.parent.GetComponent<ToggleGroup>();
+	}
+
+	public void RemoveSelfFromAnswers()
```

Comment:
What is this?

Reply:
I've replaced the function by adding RemoveQuestion() as a listener to onClick()